### ISSUE TITLE
Use arrow key instead of tab to navigate inside a comment thread

### ISSUE THREAD

**Issue by ramya-rao-a** [microsoft/vscode-pull-request-github#123]
Testing https://github.com/Microsoft/vscode/issues/55396 on Ubuntu

- Choose a comment which has replies
- Click on it in the comment panel. Focus moves to the comment in the editor
- At this point tab moves the focus to the reply

On tab, the focus usually is meant to move to the next actionable item. The comment and its replies are more like a list which should be navigate-able using arrow keys rather than tab
